lighthouse-core/scripts/chromium-web-tests/webtests/http/tests/devtools/lighthouse/lighthouse-successful-run.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+(async function() {
+  // screenshots in content shell are flaky and NO_NAVSTART occurs on bots way more frequently than local
```

Comment:
future thought: I would love to see us be able to take a stronger stance here as we have more control over the circumstances than in Chromium test runners. we've had several regressions in this area with devtools and it'd be nice to assert *something* about these results